### DISCUSSION TITLE
New feature: Image scale/place/rotate and image per step

### DISCUSSION THREAD

**Discussion by MBillhardt** [TandoorRecipes/recipes#1425] [Ideas]
Hi,

i'm quite new to TandoorRecipes (and GitHub), so i'm not shure if this is the reight place for this post.

I'm using TandoorRecipes most with portable devices (smartphone/tablet) and it wwould be a great help if i could do some basic "image manipultion" when upload an image oder editing a recipe ... like scale, rotate an move/place an image.

I like the idea to devide an recipe into serveral steps ... put id would me even better if i could ad on (or more) images to each step.

Just two ideas ...
Thanks for the great work you're doing!

Warm regards
Billy

**Comment by vabene1111** [comment]
It's planned to add some image manipulation tools but that's likely a very far future.

Adding images to steps works already using the files system. Should be rather straight forward in the latest version.
